app/src/main/java/com/ataulm/whatsnext/FilmRating.kt

```diff
@@ -0,0 +1,23 @@
+package com.ataulm.whatsnext
+
+enum class FilmRating(private val value: Float) {
```

Comment:
The API only lets us pass these specific values.

The float `value` is useful because we can convert the float from RatingBar to a FilmRating and also whatever we get from the API responses. This value is not used when sending film ratings _to_ the API.